erfa/ufunc.c.templ

```diff
@@ -207,7 +219,7 @@ static void ufunc_loop_{{ func.pyname }}(
    {#- /* if needed, allocate memory for contiguous eraLDBODY copies */ #}
    {%- if func.user_dtype == 'dt_eraLDBODY' %}
     if (copy_b) {
-        _b = PyArray_malloc(nb * sizeof(eraLDBODY));
+        _b = malloc(nb * sizeof(eraLDBODY));
```

Comment:
I guess I am slightly confused: `PyArray_malloc` is from numpy (https://numpy.org/devdocs/reference/c-api/array.html#memory-management), not python. Is the problem that that macro uses something that is not in the limited API? Probably worth adding a comment here.

Reply:
Yeah, PyArray_malloc is an alias of PyMem_RawMalloc which is not in the Python limited API (see PR description)